lib/core/src/lib/datatable/components/datatable/datatable.component.html

```diff
@@ -106,7 +112,13 @@
                         {{ getSortLiveAnnouncement(col) | translate: { string: col.title | translate } }}
                     </span>
 
-                    <span *ngIf="!col.title && !col.sortable && !headerFilterTemplate" [attr.title]="'ADF-DATATABLE.ACCESSIBILITY.EMPTY_HEADER' | translate"></span>
+                    <span
```

Comment:
Why did you remobe ngIf from this element? This will cause a bug and seems like a breaking change